python/mdvtools/project_router.py

```
@@ -163,47 +162,82 @@
+                
+                logger.info(f"Enable_auth: '{ENABLE_AUTH}'")
+                project_permissions = None
+                # Check if authentication is enabled
+                if ENABLE_AUTH:
+                    logger.info("Auth Enabled")
 
+                    user = session.get('user')
+                    user_id = user["id"] if user else None
+                    if not user_id:
+                        return jsonify({"error": "Unable to retrieve user ID"}), 401
+                    logger.info(f"Authenticated user ({auth_method}): {user_id}") 
```

Ruff (0.8.2)

Undefined name `auth_method`

(F821)